demo/server/src/app.ts

```diff
@@ -14,6 +14,16 @@ import appConfig from './util/config';
 
 const app: express.Express = express();
 const config = appConfig.getConfig();
+let corsConfig = {};
+if (!config.keycloakConfig) {
+  const dynamicOrigin = function(origin, callback) {
```

Comment:
@wtrocki @paolobueno Unfortunately '*' as origin is not allowed as I've set the credentials to be true in order to have the cookie in the requests. This was the reason I've put in this custom function in order to still allow cross-domain requests while allowing the credentials to be set to true.